packages/react/roving-focus/src/RovingFocusGroup.tsx

```diff
@@ -72,15 +72,15 @@ RovingFocusGroup.displayName = GROUP_NAME;
 /* -----------------------------------------------------------------------------------------------*/
 
 type RovingFocusGroupImplElement = React.ElementRef<typeof Primitive.div>;
-type RovingFocusGroupImplProps = Radix.MergeProps<
-  Radix.ComponentPropsWithoutRef<typeof Primitive.div>,
-  RovingFocusGroupOptions & {
-    currentTabStopId?: string | null;
-    defaultCurrentTabStopId?: string;
-    onCurrentTabStopIdChange?: (tabStopId: string | null) => void;
-    onEntryFocus?: (event: Event) => void;
-  }
->;
+type PrimitiveDivProps = Radix.ComponentPropsWithoutRef<typeof Primitive.div>;
+interface RovingFocusGroupImplProps
+  extends Omit<PrimitiveDivProps, 'dir'>,
```

Comment:
That's interesting, because that `dir` one would kinda go unnoticed previously.

Reply:
Yep, I've found that the interfaces really make you think more!